files/galaxy/dynamic_job_rules/pawsey/total_perspective_vortex/tools.yml

```diff
@@ -1231,6 +1231,30 @@ tools:
   toolshed.g2.bx.psu.edu/repos/iuc/spades_biosyntheticspades/spades_biosyntheticspades/.*:
     inherits: toolshed.g2.bx.psu.edu/repos/nml/spades/spades/.*
 
+  # Stacks 2 tools
+  toolshed.g2.bx.psu.edu/repos/iuc/stacks2_clonefilter/stacks2_clonefilter/*:
```

Comment:
needs to be /.*

Reply:
@Slugger70 glob will not work for matching, these needs to be `toolshed.g2.bx.psu.edu/repos/iuc/stacks2_clonefilter/stacks2_clonefilter/.*` to match the tool ids

Reply:
ahhh.. ok